src/main/java/mate/academy/bookstore/dto/OrderItemRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.bookstore.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class OrderItemRequestDto {
+    @NotNull
+    @Positive
+    private Long bookId;
+    @NotNull(message = "Please enter quantity ")
+    @Min(value = 0, message = "Invalid quantity, it cannot be less than zero")
+    private int quantity;
```

Comment:
It is better to use the Class
```suggestion
    private Integer quantity;
```